src/main/java/io/github/mirromutth/r2dbc/mysql/collation/CharCollations.java

```diff
@@ -73,7 +75,7 @@ static CharCollation fromId(int id, ServerVersion version) {
                 }
             }
 
-            if (version.compareTo(8, 0, 1) >= 0) {
+            if (version.isGreaterThanOrEqualTo(UTF8MB4_0900_VER)) {
```

Comment:
@mp911de Is it probably like this commit, a commit only does one thing? I will try to make the subsequent commits like this.

Reply:
A single commit should have a single topic/concern that it is touching.